WDACConfig/WDACConfig Module Files/Shared/New-SnapBackGuarantee.psm1

```diff
@@ -0,0 +1,75 @@
+Function New-SnapBackGuarantee {
+    <#
+    .SYNOPSIS
+        A function that arms the system with a snapback guarantee in case of a reboot during the base policy enforcement process.
+        This will help prevent the system from being stuck in audit mode in case of a power outage or a reboot during the base policy enforcement process.
+    .PARAMETER Location
+        The directory location of the base policy file that will be enforced.
+    .INPUTS
+        System.IO.DirectoryInfo
+    .OUTPUTS
+        System.Void
+    #>
+    [CmdletBinding()]
+    Param(
+        [parameter(Mandatory = $true)]
+        [System.IO.DirectoryInfo]$Location
```

Comment:
This parameter needs to be renamed to `Path` according to PowerShell best pracitces.